pkgs/top-level/python-packages.nix

```
@@ -1114,7 +1114,7 @@ in {
 
   pydy = callPackage ../development/python-modules/pydy { };
 
-  pyexiv2 = disabledIf isPy3k (toPythonModule (callPackage ../development/python-modules/pyexiv2 {}));
+  pyexiv2 = disabledIf isPy27 (toPythonModule (callPackage ../development/python-modules/pyexiv2 {}));
```

and move disabled logic into expression

toPythonModule isn't needed when using `buildPythonPackage`
```suggestion
  pyexiv2 = callPackage ../development/python-modules/pyexiv2 {};
```